src/tabmanager.cc

```diff
@@ -101,6 +101,44 @@ void TabManager::closeTabRequested(int x)
     delete temp;
 }
 
+void TabManager::closeCurrentTab()
+{
+    assert(tabWidget != nullptr);
+
+    /* Close tab or close the current window if only one tab is open. */
+    if (tabWidget->count()>1)
+        this->closeTabRequested(tabWidget->currentIndex());
+    else
+        par->close();
+}
+
+void TabManager::nextTab()
+{
+    assert(tabWidget != nullptr);
+
+    if (tabWidget->count()<=1)
+        return;
+
+    int idx = (tabWidget->currentIndex()+1) % tabWidget->count();
+    PRINTB("NextTab, new-idx=%d", idx);
```

Comment:
Please remove the output or convert to debug (`PRINTDB`) so it's not shown without being specifically enabled via `--debug`.